### ISSUE TITLE
Response variable scaled when iScalCol=2

### ISSUE THREAD

**Issue by miguelcorrea** [enanomapper/RRegrs#8]
iSCalCol=2 is supposed to only scale the features.

**Comment by egonw** [comment]
@miguelcorrea can you give me some detail in person today?

**Comment by cafernandezlo** [comment]
Fixed in Release Version 0.0.5

```
  # if NORMALIZATION
  if (s==1) {
    # Scale all the features (from column c; column 1 is the predictor output)
    if(c==2){
      maxs <- apply(ds[c:ncol(ds)], 2, max)
      mins <- apply(ds[c:ncol(ds)], 2, min)
      ds.norm.scale<-scale(ds[c:ncol(ds)], center = mins, scale = maxs - mins)
      DataSet.scaled<-cbind(ds[,1],ds.norm.scale)
    }else{
      maxs <- apply(ds, 2, max)
      mins <- apply(ds, 2, min)
      DataSet.scaled<-scale(ds, center = mins, scale = maxs - mins)
    }

  }
  # if STADARDIZATION
  if (s==2) {
    # Scale all the features (from column c; column 1 is the predictor output)
    if(c==2){
      DataSet.scaled <- scale(ds[c:ncol(ds)],center=TRUE,scale=TRUE)
      DataSet.scaled<-cbind(ds[,1],DataSet.scaled)
    }else{
      DataSet.scaled<-scale(ds,center=TRUE,scale=TRUE)
    }
  }
```
